lib/handlers/get.js

```diff
@@ -128,6 +128,9 @@ async function handler (req, res, next) {
     res.send(data)
     return next()
   } catch (err) {
+    if (err.message === '404') {
+      return next(error(404, 'HTML do not contain data island'))
```

Comment:
```suggestion
      return next(error(404, 'HTML does not contain any data islands'))
```

Reply:
The plural in `islands` seems to allow multiples. Actually my PR only allows one.
Not sure that more than one could be expected. The `id="data"` cannot be used more than once in an html document.

Reply:
There may be multiple islands in a single HTML doc, tho perhaps not (yet) in your environment. My error text is equivalent to saying, "I found no data island in this HTML," among other phrasings.

Another phrasing suggestion might be, `HTML does not contain a data island`.